styles/_base.css

```diff
@@ -23,6 +23,10 @@ h2 {
   font-size: 20px;
 }
 
+main {
+  overflow: scroll;
+}
+
 input,
 button {
```

Comment:
Is this the button you use for the tags in your `quiz-card` then you should style it in the component and give it a BEM class.